### PR TITLE
Installing Keptn CLI document enhanced

### PR DESCRIPTION
**Pull request by NitishKumar06**
Fix: #1221

The following guide on how to install Keptn CLI (https://keptn.sh/docs/quickstart/) doesn't contain the better explanation as it is mentioned here:https://tutorials.keptn.sh/tutorials/keptn-full-tour-prometheus-07/index.html?index=..%2F..keptn07x#3

So after looking at it, the following changes have been made to improve the documentation on how to install Keptn CLI onto your OS.

### CODE REVIEW

File: content/docs/quickstart/_index.md [hunk 1]
```diff
@@ -77,13 +77,41 @@ You can also run additional `hello` sequences with: `keptn trigger sequence --se
...
+keptn --help
+```
 
-2) Install Keptn core control plane and execution plane services for continuous delivery using the CLI:
+- Windows
+  
+ ```
+.\keptn.exe --help
+ ```
+ Install Keptn core control plane and execution plane services for continuous delivery using the CLI:
```
Comment (oleg-nenashev) [inline content/docs/quickstart/_index.md:114]: A separator here would be needed

**Review by oleg-nenashev — COMMENTED** [review]
LGTM overall, just one minor thing
@StackScribe please take a look

File: content/docs/quickstart/_index.md (hunk 1 above)
Comment (NitishKumar06) [reply content/docs/quickstart/_index.md:114]: @StackScribe ?

**Comment by aepfli** [comment]
@NitishKumar06  thank you for this contribution, but sadly, I think this was somewhat superseded by https://github.com/keptn/keptn.github.io/issues/1509 and can therefore be closed - @StackScribe, or am I wrong here :)

**Comment by thisthat** [comment]
Closing because of superseded by #1509

**Pull request closed without merging**